GOVERNANCE.md

```diff
@@ -42,6 +45,33 @@ Maintainers have right to nominate contributors to become maintainers and vote o
 
 Individuals can be maintainers to multiple Eiffel Community Open Source Projects.
 
+
+## Community Roles
+
+Eiffel Community currently has two roles relevant to community governance and support.
+
+### Election Officers
+
+Eiffel Community Election Officers are the community members who are appointed by Eiffel Technical Committee to conduct community elections.
+
+The term for Election Officers is **one year**.
+
+Responsibilities of Election Officers are
+
+* Determine the list of community members who are eligible to vote and run in community elections.
+* Conduct the elections based on the process documented in [Election Process](#Election-Process).
+* Ensure the elections are done in a transparent way.
+* Guide the Technical Committee and Eiffel Community at large in election matters.
+
+Eiffel Community Election Officers are the individuals listed below:
```

Comment:
Should we state the number of election officers (2?) or is that up to the TC to decide ad-hoc?

Reply:
I think we should. Will amend the PR and state 2 as number of election officers.

This reminds me something else so just highlighting this.
Normally communities appoint people as officers who do not have a stake in the elections they are handling in order to ensure there is no conflict of interest.
In our case, we don't need to follow this given the size of the community but still good for everyone to be aware of how elections are conducted and by whom.